rclcpp/include/rclcpp/node_interfaces/node_graph.hpp

```diff
@@ -61,6 +61,11 @@ class NodeGraph : public NodeGraphInterface
 
   RCLCPP_PUBLIC
   virtual
+  std::vector<std::string>
```

Comment:
We can keep it as-is for now. We can always revise it if necessary in the future.